pkg/api/steve/catalog/types/rest.go

```diff
@@ -47,6 +47,7 @@ type ChartUpgradeAction struct {
 	DisableHooks             bool             `json:"noHooks,omitempty"`
 	DisableOpenAPIValidation bool             `json:"disableOpenAPIValidation,omitempty"`
 	Force                    bool             `json:"force,omitempty"`
+	ForceAdopt               bool             `json:"forceAdopt,omitempty"`
```

Comment:
This exists just to adopt the system-charts install right ?

Reply:
Yes, for upgrade case, to adopt resources that were created by helm 2.